cmake/configs/posix_sitl_default.cmake

```diff
@@ -104,7 +104,7 @@ set(config_extra_builtin_cmds
 # for the config posix_sitl_efk2 and set again, explicitly, for posix_sitl_lpe,
 # which are based on posix_sitl_default.
 set(config_sitl_rcS_dir
-	posix-configs/SITL/init/lpe
+	posix-configs/SITL/init/ekf2
```

Comment:
We're using EKF2 as the global default for all PX4 powered vehicles. On-vehicle and SITL should be consistent.